src/OrleansRuntime/Scheduler/WorkerPoolThread.cs

```diff
@@ -102,8 +103,11 @@ internal WorkerPoolThread(WorkerPool gtp, OrleansTaskScheduler sched, int thread
                 WorkerThreadStatisticsNumber = SchedulerStatisticsGroup.RegisterWorkingThread(Name);
         }
 
+        private readonly ActionBlock<IWorkItem> _executor;
         protected override void Run()
         {
+            // not used anymore
+            Thread.Sleep(int.MaxValue);
```

Comment:
>some todo's, which will be addressed after Orleans internal performance test run, which will also show whether it worthy reviewing

As I  previously mentioned this PR needs some polishing which won't affect functionality. I wanted to see full sized performance test results first in order to see where it's going to go.